### DISCUSSION TITLE
Session 4 Exercise, Q3

### DISCUSSION THREAD

**Discussion by DikiDL** [LiNk-NY/IntroR#39] [General]
<img width="460" alt="Screenshot 2024-06-08 at 7 26 30 PM" src="https://github.com/CUNYSPHCode/IntroR/assets/170993332/d288174e-654e-4e70-ab73-88ab482bf0b2">

For Q3 on the exercise, although this was discussed briefly in the live lecture, I still don't quite understand how to read the graph from the pairs.panel output. Any ideas? Thank you!

**Comment by ralythompson24** [comment]
The intersection between any two variables shows the correlation between those two variables. On the top right corner, these are the correlation coefficient values, and on the bottom left are scatterplots showing the two variables crossed with one another. For example, the correlation between disp and hp is 0.79. Does that help?

**Reply by abhuiyan2015** [reply]
Thank you for the clarification. I had also thought about this as well.

**Comment by ketoon43** [comment]
This was helpful! Thank you very much!
